<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup Watershed data</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Surface topographic layers</string>
   </property>
   <widget class="QComboBox" name="cmbWatershedArea">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbWatershedSlope">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbFdir">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>331</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Flow direction index type</string>
    </property>
    <widget class="QRadioButton" name="rbFDIndexTypeNE">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start from northeast</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbFDIndexTypeN">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start from north</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdioEAST">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>East</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdioTaudem">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Taudem index</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="cmbFac">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Flow accumulation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>30</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Watershed area:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Watershed slope:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Flow direction:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>371</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>Channel layer</string>
   </property>
   <widget class="QComboBox" name="cmbStream">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>210</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbChannelWidthLayer">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>210</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkStreamLayer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>39</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>          Stream:</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkChannelWidthLayer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>74</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel width:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>587</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>677</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>371</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Initial condition</string>
   </property>
   <widget class="QComboBox" name="cmbiniSoilRatioLayer">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>210</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkiniSoilRatioLayer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ini. soil sat. ratio:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkIniFlow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ini. channel flow</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbiniFlowLayer">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>210</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
